src/main/index.dev.ts

```diff
@@ -8,7 +8,7 @@
 /* eslint-disable */
 
 // Set environment for development
-process.env.NODE_ENV = "development";
+// process.env.NODE_ENV = "development";
```

Comment:
Why was this commented out?

Reply:
The features of mode have been added to webpack 4. `process.env.NODE_ENV = "development";` is not necessary

Reply:
Great. Please delete that block then :)